src/app/pages/users/user-sites/user-sites-site-admin.component.ts

```diff
@@ -7,23 +7,25 @@ import { MessageService } from '../../../services/message.service';
 import { CellContentTemplateDirective } from '../../../shared/table/cell-content-template/cell-content-template.directive';
 import { RestResponse } from '../../../types/GlobalType';
 import { UserSite } from '../../../types/Site';
-import { User, UserToken } from '../../../types/User';
+import { User, UserRole, UserToken } from '../../../types/User';
 import { Utils } from '../../../utils/Utils';
 
 @Component({
   template: `
     <div class="d-flex justify-content-center">
       <mat-checkbox class="mx-auto"
-        [checked]="(row.siteAdmin ? row.siteAdmin : false)"
-        [disabled]="(!loggedUser.sitesAdmin.includes(row.site.id) && loggedUser.role !== 'A')"
+        [checked]="row?.siteAdmin || user?.role === UserRole.ADMIN"
+        [disabled]="(!isSiteAdmin && loggedUser.role !== UserRole.ADMIN) || columnDef.disabled || user?.role === UserRole.ADMIN"
```

Comment:
Hi @LucasBrazi06, I don't understand why the Admin cannot be set as siteAdmin ?
Imagine siteAdmin (extended from basic) has new specific rights (eg: refund) the Admin won't have those new rights automatically, right ? They are not linked together as far as I know. How is it different from siteOwner ?

Reply:
@cabch Sorry for the late answer.
Site Admin for an Admin means nothing as he's already admin of all the Sites!
Site Admin is an elevation of privilege for Basic users on a given Site. 
Refund is only allowed to users with role Site Owner and an Admin can be Site Owner.
Voila, t'as compris ou je viens au Labs t'expliquer lol :-) !